src/domain/johtoselvitys/Contacts.test.tsx

```diff
@@ -1,15 +1,22 @@
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
 import React from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
 import { fireEvent, render, screen } from '../../testUtils/render';
 import { Contacts } from './Contacts';
 import hankkeet from '../mocks/data/hankkeet-data';
+import applications from '../mocks/data/hakemukset-data';
 import { HankeContact, HankeDataDraft } from '../types/hanke';
+import { JohtoselvitysFormValues } from './types';
 
 jest.setTimeout(10000);
 
-function Form({ hanke }: { hanke?: HankeDataDraft }) {
-  const methods = useForm({});
+function Form({
+  hanke,
+  application,
+}: {
+  hanke?: HankeDataDraft;
+  application?: JohtoselvitysFormValues;
+}) {
+  const methods = useForm({ defaultValues: application || { applicationData: {} } });
```

Comment:
Sonar suggests to replace || with ?? as it uses the default value only on null or undefined. It might not be a big of a deal here but anyway.

Reply:
Made the change.